Tests/otlLib/mock_builder_test.py

```diff
@@ -0,0 +1,90 @@
+from fontTools.otlLib.builder import (
+    AlternateSubstBuilder,
+    ChainContextPosBuilder,
+    ChainContextSubstBuilder,
+    LigatureSubstBuilder,
+    MultipleSubstBuilder,
+    CursivePosBuilder,
+    MarkBasePosBuilder,
+    MarkLigPosBuilder,
+    MarkMarkPosBuilder,
+    ReverseChainSingleSubstBuilder,
+    SingleSubstBuilder,
+    ClassPairPosSubtableBuilder,
+    PairPosBuilder,
+    SinglePosBuilder,
+)
+from fontTools.otlLib.error import OpenTypeLibError
+from fontTools.ttLib import TTFont
+from fontTools.misc.loggingTools import CapturingLogHandler
+import logging
+import unittest
+
+
+def makeTTFont():
```

Comment:
in pytest, this would be a ["fixture"](https://docs.pytest.org/en/stable/fixture.html) function

```python
@pytest.fixture
def ttfont():
    ...
```

then the test methods that need this would add a parameter with the name of that fixture, e.g.:

```python
def test_something(ttfont):
    assert "a" in ttfont.getGlyphOrder()
```